components/support/error/Cargo.toml

```diff
@@ -24,3 +24,6 @@ uniffi = { version = "0.28.2", features=["build"]}
 # Temp hack for uniffi 0.28 generated code, will not be necessary after next uniffi update.
 [lints.clippy]
 empty-line-after-doc-comments = "allow"
+
+[features]
```

Comment:
this one surprised me - the error crate had a cfg gate for a non-defined feature. Due to the use of macros, this feature also bleeds into rc_crypto. I think we can probably kill this feature as it's (obviously) unused, but I didn't want to do that here at this time (although open to suggestions I should)